source/CommonJobs/CommonJobs.Infrastructure.RavenDb/Command.cs

```diff
@@ -3,7 +3,7 @@
 using System.Linq;
 using System.Web;
 using Raven.Client;
-using Newtonsoft.Json;
+using Raven.Imports.Newtonsoft.Json;
```

Comment:
We should enforce the use of Raven's own newtonsoft json (raven has a copy of newtonsoft.json to avoid some known issues). So I will probably remove the plain Newtonsoft.json lib from where it's not needed.